src/SeisSol.cpp

```diff
@@ -184,6 +186,8 @@ void seissol::SeisSol::finalize()
 
 	const int rank = MPI::mpi.rank();
 
+	m_timeManager.resetCommunicationManager();
+
```

Comment:
Not sure why this is necessary.

Reply:
Ah that makes sense. But why don't you call it freeCommunicationManager if that's the purpose?